scss/components/_header.scss

```diff
@@ -0,0 +1,274 @@
+.main-header {
+    background: $headerColor;
+    padding: 15px 0;
+    z-index: 100;
+    position: relative;
+    &__inner {
+        display: flex;
+        justify-content: space-between;
+    }
+
+    .container {
+        &:before,
+        &:after {
+            content: none;
+        }
+    }
+
+    &__left {
+        display: flex;
+        align-items: center;
+    }
+
+    &__right {
+        display: flex;
+        align-items: center;
+        padding: 0 0 0 15px;
+    }
+
+    &__logo {
+        display: block;
+        width: 156px;
+        margin: 0 24px 0 0;
+
+        img {
+            display: block;
+            width: 100%;
+            height: auto;
+        }
+    }
+
+    .button {
+        .icon-rss {
+            font-size: 16px;
+        }
+    }
+
+    &__search {
+        position: relative;
+        z-index: 3;
+
+        & > .icon-search {
+            position: absolute;
+            z-index: 4;
+            pointer-events: none;
+            color: $grey;
+            font-size: 16px;
+            top: 15px;
+            left: 5px;
+        }
+
+        input[type="text"],
+        input[type="search"] {
+            display: block;
+            height: 46px;
+            padding: 12px 40px;
+            border: 1px solid transparent;
+            border-bottom-color: #ebebeb;
+            font-size: 16px;
+            line-height: 20px;
+            font-weight: 500;
+            color: $grey;
+            background: none;
+            outline: none;
+            outline-width: 0;
+            letter-spacing: .02em;
+            transition: color .3s linear, background .3s linear, border-color .3s linear;
+
+            &:focus {
+                border-bottom-color: $black;
+            }
+        }
+
+        .algolia-autocomplete {
+            display: block !important;
+        }
+    }
+}
+
+.main-nav {
+    margin: 0 24px 0 0;
+    position: relative;
+
+    &:hover {
+        & > .main-nav__drop {
+            display: block;
+        }
+
+        & > .main-nav__opener {
```

Comment:
Why do you need `>` here?

Reply:
there are a few links in the multilevel menu